gui/syncthing/usagereport/usageReportPreviewModalView.html

```diff
@@ -1,17 +1,5 @@
-<div id="urPreview" class="modal fade" tabindex="-1">
-	<div class="modal-dialog modal-lg">
-	  <div class="modal-content">
-	    <div class="modal-header alert alert-success">
-	      <h4 translate class="modal-title">Anonymous Usage Reporting</h4>
-	    </div>
-	    <div class="modal-body">
-	      <p translate>The encrypted usage report is sent daily. It is used to track common platforms, folder sizes and app versions. If the reported data set is changed you will be prompted with this dialog again.</p>
-	      <p translate translate-value-url="<a href=&quot;https://data.syncthing.net&quot; target=&quot;_blank&quot;>https://data.syncthing.net</a>">The aggregated statistics are publicly available at {%url%}.</p>
-	      <pre><small>{{reportData | json}}</small></pre>
-	    </div>
-	    <div class="modal-footer">
-	      <button type="button" class="btn btn-success btn-sm" data-dismiss="modal"><span class="glyphicon glyphicon-ok"></span>&nbsp;<span translate>OK</span></button>
-	    </div>
-	  </div>
-	</div>
-</div>
+<modal id="urPreview" status="success" icon="bar-chart" title="{{'Anonymous Usage Reporting' | translate}}" large="yes" close="yes" tabindex="-1">
+  <p translate>The encrypted usage report is sent daily. It is used to track common platforms, folder sizes and app versions. If the reported data set is changed you will be prompted with this dialog again.</p>
+  <p translate translate-value-url="<a href=&quot;https://data.syncthing.net&quot; target=&quot;_blank&quot;>https://data.syncthing.net</a>">The aggregated statistics are publicly available at {%url%}.</p>
+  <pre><small>{{reportData | json}}</small></pre>
+</modal>
```

Comment:
whoever extraced the modals into separate files forgot about this one, so I corrected that. It is the same content but it uses the angular directive which then loads the above into modal.html which adds a close button at the bottom and the modal header - voila!